### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by 7sne**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
   env: {
     es6: true,
   },
-  extends: ['typestrict', 'react-app', 'plugin:storybook/recommended'],
```
Comment (krzkaczor) [inline .eslintrc.js:8]: why remove typestrict :(?

File: src/components/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  rawAbi: T;
+  tabState: {
+    tab: 'abi' | '4-bytes';
+    setTab: Dispatch<SetStateAction<'abi' | '4-bytes'>>;
+  };
+  setDecodeResults: Dispatch<React.SetStateAction<D>>;
+  handleChangeRawAbi: (e: ChangeEvent<HTMLTextAreaElement>) => void;
+}
+
+export function Tabs<
```
[the remark this answers is not in the document]
Comment (7sne) [reply src/components/Tabs.tsx:20]: I don't think that there's a need to make everything a generic component. It's a different story when we would plan to create our component library specifically, but for that use case, the current state is all right IMO

[the remark this answers is not in the document]
Comment (7sne) [reply src/components/Tabs.tsx:20]: And yeah, the name is misleading now indeed, I will change it.

File: pages/tx-decoder.page.tsx [hunk 3]
```diff
@@ -17,8 +17,10 @@ export default function TxDecoder() {
   const [rawTx, setRawTx] = useState<WithOkAndErrorMsgOptional<string>>({
     isOk: true,
   });
-  const [decodeResults, setDecodeResults] =
-    useState<{ tx: TypedTransaction; senderAddr: string }>();
+  const [decodeResults, setDecodeResults] = useState<{
```
[the remark this answers is not in the document]
Comment (7sne) [reply pages/tx-decoder.page.tsx:20]: Hmm. We had that bug in another project and I think that sometimes on a given length prettier tends to bug, I will investigate that.

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**